.travis.yml

```diff
@@ -31,6 +31,7 @@ matrix:
       before_install:
         - Rscript -e 'install.packages("devtools", dependencies = TRUE)'
         - cd $TRAVIS_BUILD_DIR/src/r
+        - touch NAMESPACE
```

Comment:
Also since we did not commit `RcppExports.R`, when fresh installing, `devtools::document()` sometimes will not run `Rcpp::compileAttributes()` to automatically create it. It is safe to call it after `devtools::document()`.